src/assets/Repeater_Peashooter.java

```diff
@@ -0,0 +1,49 @@
+package assets;
+
+import engine.Board;
+import util.Logger;
+
+public class Repeater_Peashooter extends Plant{
+private static Logger LOG = new Logger("Repeated Peashooter");
+	
+	private static final int DEFAULT_HP = HEALTH_MEDIUM;
+	private static final int DEFAULT_POWER = ATTACK_LOW * 2;
```

Comment:
Use the provided constants. None of this "+magic number" or "*magic number" stuff